tests/countries/test_falkland_islands.py

```diff
@@ -0,0 +1,228 @@
+        obs_dt = (
+            "1983-08-15",
+            "1988-08-15",
+            "1993-08-16",
+            "1994-08-15",
+            "1999-08-16",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoHolidayName(name, range(2002, 2050))
+        self.assertHolidayName(
+            name, self.workday_holidays, (f"{year}-08-14" for year in range(2002, 2050))
+        )
+        self.assertNoHolidayName(name, self.workday_holidays, range(1983, 2002))
+
+    def test_peat_cutting_day(self):
+        name = "Peat Cutting Day"
+        self.assertHolidayName(
+            name,
+            "2020-10-05",
+            "2021-10-04",
+            "2022-10-03",
+            "2023-10-02",
+            "2024-10-07",
+            "2025-10-06",
+        )
+        self.assertHolidayName(name, range(2002, 2050))
+        self.assertNoHolidayName(name, range(1983, 2002))
+
+    def test_kings_birthday(self):
+        name = "HM The King's Birthday"
+        self.assertHolidayName(name, (f"{year}-11-14" for year in range(2022, 2050)))
+        obs_dt = (
+            "2026-11-16",
+            "2027-11-15",
+            "2032-11-15",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+        self.assertNoHolidayName(name, range(1983, 2022))
+
+    def test_christmas_day(self):
+        name = "Christmas Day"
+        self.assertHolidayName(name, (f"{year}-12-25" for year in range(1983, 2050)))
+        obs_dt = (
+            "2010-12-28",
+            "2011-12-28",
+            "2016-12-28",
+            "2021-12-28",
+            "2022-12-28",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+
+    def test_boxing_day(self):
+        name = "Boxing Day"
+        self.assertHolidayName(name, (f"{year}-12-26" for year in range(1983, 2050)))
+        obs_dt = (
+            "2010-12-29",
+            "2015-12-28",
+            "2020-12-28",
+            "2021-12-29",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+
+    def test_christmas_day_three(self):
+        name = "Christmas Holiday"
+        self.assertHolidayName(name, (f"{year}-12-27" for year in range(1983, 2050)))
+        obs_dt = (
+            "2014-12-29",
+            "2015-12-29",
+            "2020-12-29",
+            "2025-12-29",
+        )
+        self.assertHolidayName(f"{name} (observed)", obs_dt)
+
+    def test_government_holiday(self):
+        name = "Government Holiday"
+        self.assertHolidayName(
+            name,
+            self.government_holidays,
+            "2017-12-28",
+            "2017-12-29",
+            "2018-12-28",
+            "2022-12-29",
+            "2023-12-28",
+            "2023-12-29",
+            "2024-12-30",
+            "2024-12-31",
+            "2025-12-30",
+            "2025-12-31",
+        )
+        self.assertHolidayName(name, self.government_holidays, range(1983, 2050))
```

Comment:
Need to make sure that government holidays aren't included in `PUBLIC` list:
```suggestion
        self.assertHolidayName(name, self.government_holidays, range(1983, 2050))
        self.assertNoHolidayName(name)
```